docs/shared_vip.md

```diff
@@ -0,0 +1,100 @@
+AKO provides support for sharing VIP among multiple kubernetes Services of type LoadBalancer. Generally, with LoadBalancer Services, AKO creates dedicated L4 Virtual Services in the Avi Controller, but multiple LoadBalancer Services can also be combined to share a single VIP.
+
+> **Note**: The ability to share a single VIP among multiple LoadBalancer Services is released as tech-preview as part of AKO release-1.7.1
+
+This can be achieved by providing an annotation to multiple LoadBalancer Services, where VIP sharing is intended.
+The annotation to be applied is `ako.vmware.com/enable-shared-vip` with a string value. An example of this is shown below.
```

Comment:
We can combine this two sentences `The annotation to be applied is `ako.vmware.com/enable-shared-vip` with a string value. An example of this is shown below.` as `The annotation to be applied is `ako.vmware.com/enable-shared-vip` with a string value as shown below`